framework/auth/__init__.py

```diff
@@ -55,7 +75,6 @@ def external_first_login_authenticate(user, response):
         'auth_user_external_id_provider': user['external_id_provider'],
         'auth_user_external_id': user['external_id'],
         'auth_user_fullname': user['fullname'],
-        'auth_user_access_token': user['access_token'],
```

Comment:
Similarly, don't store this access token (no longer exists) in the session data.